docs/qa/CheckLists/FavoritePage_tags.md

```diff
@@ -0,0 +1,68 @@
+| 6  | Text 1 line "create a tag:" 2 line "create a tag for adding translations to favorites in a well-organized way 3 line placeholder "e.g Lesson1"<br>4 line "You can always manage your tags, renaming, adding, and deleting them in the favorites section" |             |                                                                                    |
+| 7  |                                                                                                                                                                                                                                                          |             |
+| 8  | Texts are in the same format as in layout, text does not go out of screen                                                                                                                                                                                |             |                                                                                    |
+| 9  | Red button "create" is below all text                                                                                                                                                                                                                    |             |                                                                                    |
+| 10 | Above text is a signature image of stars                                                                                                                                                                                                                 |             |                                                                                    |
+| 11 | Validation? For example no # allowed                                                                                                                                                                                                                     |             | perhaps check for sql injections, max number of string [e.t](http://e.tc/).c?      |
+| 12 | If this is the first tag of user, switch to 13_favourites_first_tag, else switch to page 14                                                                                                                                                              |             |                                                                                    |
+|    | 1.3 First tag created 13                                                                                                                                                                                                                                 |             | here a limit on how many tags users can create?                                    |
```

Comment:
I would move creating first tag section before the favourites tag creation one, but it's up to you